homeassistant/components/chacon_dio/entity.py

```diff
@@ -51,3 +51,12 @@ def callback_device_state(self, data: dict[str, Any]) -> None:
         _LOGGER.debug("Data received from server %s", data)
         self._update_attr(data)
         self.async_write_ha_state()
+
+    async def async_update(self) -> None:
+        """Update the entity : called when using the action homassistant.update_entity."""
+
+        _LOGGER.debug("Update called for %s, %s", self, self.target_id)
+        data = await self.client.get_status_details([self.target_id])
+        _LOGGER.debug("Received data from server %s", data)
+        self._update_attr(data[self.target_id])
+        self.async_write_ha_state()
```

Comment:
Please double check if the state write is needed, it might be so that this is built in. Otherwise we're doing 2 state writes

Reply:
You were right, the async_write_ha_state is not necessary. I removed it.